packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -27,7 +29,15 @@ const styles = {
 };
 
 const GlobalHeader = ({ editUrl, className, search }) => {
-  const [isOverlayOpen, setIsOverlayOpen] = useState(false);
+  const location = useLocation();
+  const [isOverlayOpen, setIsOverlayOpen] = useState(
+    Boolean(qs.parse(location.search).q)
+  );
+
+  useEffect(() => {
+    if (isOverlayOpen && !qs.parse(location.search).q) navigate('/?q=');
+    if (!isOverlayOpen) navigate('/');
```

Comment:
Does this mean once the overlay is closed, it will always navigate back to the home page? If so, we should probably try and keep the user on the same page, just remove the query params.